docs/cloud_providers/aws/index.md

```diff
@@ -0,0 +1,76 @@
+# cloud-provider-aws
+
+The AWS cloud provider provides the interface between a
+Kubernetes cluster and AWS service APIs.
+This project allows a Kubernetes cluster to provision,
+monitor and remove AWS resources necessary for operation of the cluster.
+
+[See the official online documentation here](https://cloud-provider-aws.sigs.k8s.io)
+
+## k0s specific instructions
+
+**Before doing steps above, follow the [official prerequisites guide](https://cloud-provider-aws.sigs.k8s.io/prerequisites/)**
+
+## Cluster installation
+
+### k0sctl installation
+
+Use following k0sctl config as a starting point
+
+```yaml
+apiVersion: k0sctl.k0sproject.io/v1beta1
+kind: Cluster
+metadata:
+  name: k0s-cluster
+spec:
+  hosts:
+  - role: controller
+    ssh:
+      address: 10.0.0.1 # replace with the controller's IP address
+      user: root
+      keyPath: ~/.ssh/id_rsa
+  - role: worker
+    ssh:
+      address: 10.0.0.2 # replace with the worker's IP address
+      user: root
+      keyPath: ~/.ssh/id_rsa
+      installFlags:
+        - --enable-cloud-provider
+        - --labels node-role.kubernetes.io/master=""
+```
+
+### Manual install
+
+#### Controller install
+
+Follow the [guide for controller installation](../install.md)
```

Comment:
k0s-multi-node.md could be a better reference as install.md is for single node use.